pkg/kafka/consumer/reader.go

```diff
@@ -44,7 +44,7 @@ func NewReader(
 
 		// Topic.
 		Topic:                 conf.FQTopic,
-		GroupID:               conf.GroupID,
+		GroupID:               conf.FQGroupID,
```

Comment:
I would like to rename the conf variable to settings, as we usually name it settings.